### ISSUE TITLE
Replicating MAttNet Online Demo

### ISSUE THREAD

**Issue by albamhp** [lichengunc/MAttNet#28]
Running the demo `cv/example_demo.ipynb` with the original configuration and images from DAVIS2017 challenge and captions, give significantly different results than the Online Demo.

For example, the first frame from `girl-dog` sequence the following candidates are obtained:
![image](https://user-images.githubusercontent.com/17745828/59370785-8e39f900-8d43-11e9-97af-b67421c84063.png)

An example using `a black dog` as caption is showed below:

ONLINE DEMO:
![image](https://user-images.githubusercontent.com/17745828/59371102-60a17f80-8d44-11e9-98a1-3ba08e69e290.png)

Using RefCOCO Pre-trained model weights, the results using `cv/example_demo.ipynb`  with different captions for dog don't seem to change, they always return the girl, while the online demo works fine.

EXAMPLE DEMO 
![image](https://user-images.githubusercontent.com/17745828/59371142-7dd64e00-8d44-11e9-94f0-90bec892d828.png)

I tried with the weights provided with several captions. RefCOCO+ works for dog but not wheelchair captions, while RefCOCOg works for wheelchair but not dog captions.  The results are never the same than those in the Online Demo, which works fine with all captions. 

Is there a different configuration available that replicates those results?

**Comment by lichengunc** [comment]
Yes, I merged several datasets and retrained the model for the demo use. So the output would be different from the released ones here.

**Comment by carlesventura** [comment]
Would it be possible to release the model that is used in the demo online? If this model is better than the ones released, it would be benefitial for the comunity.
